src/Components/test/E2ETest/Infrastructure/ServerFixtures/ToggleExecutionModeServerFixture.cs

```diff
@@ -32,8 +31,7 @@ protected override string StartAndGetRootUri()
         {
             if (_buildWebHostMethod == null)
             {
-                // Use Blazor's dev host server
-                var underlying = new DevHostServerFixture<TClientProgram>();
```

Comment:
Why do we need to remove the devhost server fixture?

We want in general to know that things work well with our devhost server

Reply:
Is this a rename that it's not picking up correctly?